packages/react/src/primitives/View/View.tsx

```diff
@@ -28,6 +29,7 @@ const ViewPrimitive = <Element extends ElementType = 'div'>(
       'data-testid': testId,
       disabled: isDisabled,
       ref,
+      inert: inert ? '' : null,
```

Comment:
What about this?

```suggestion
           ...(inert && { inert: "" }),
```

Reply:
also, if the type of `inert` is `boolean`, should it be boolean here?

`      ...(true && { inert: true }),`

Reply:
I guess if you use boolean value here React will render `inert=true/false` but we just want `inert` on the element

Reply:
This is actually a React issue, we can't pass the boolean `true` here or it won't actually get rendered in the DOM. See this quick example:

https://codesandbox.io/s/crazy-wu-02mz2p?file=/src/App.js

Reply:
Heather also documented this in the issue (Thanks Heather!) https://github.com/aws-amplify/amplify-ui/issues/2632